compiler/p/codegen/OMRMemoryReference.hpp

```diff
@@ -138,18 +140,13 @@ class OMR_EXTENSIBLE MemoryReference : public OMR::MemoryReference
       return false;
       }
 
+   bool isDelayedOffsetDone() { return (_flag & TR_PPCMemoryReferenceControl_DelayedOffsetDone) != 0; }
+   void setDelayedOffsetDone() { _flag |= TR_PPCMemoryReferenceControl_DelayedOffsetDone; }
```

Comment:
You're right, these APIs will be used in a future refactor. They're included here since this flag is checked in the expansion unit tests, so the tests wouldn't compile if we didn't have the getter at the very least.

I've added some Doxygen to describe what this flag is doing in 0e5b91c5dea69ec7c9a0867c39a2456b884131e3.